AnimateSvg.js

```diff
@@ -46,56 +94,16 @@ export default function AnimateSvg(props) {
         useNativeDriver: true,
       }),
     ]).start();
-    const endOfAnim = setTimeout(() => {
-      setIsRegistered(false);
-      clearTimeout(endOfAnim);
-    }, Math.max(SVG_OUT_DURATION, QUOTES_IN_DURATION) + QUOTES_IN_DELAY);
+  };
+
+  // LogoSvg fades out first time
+  if (isShakeTriggered && !hasToggledBefore && !isLogoAnimationRunning) {
```

Comment:
State variable names have been updated to more accurately reflect their purpose in code base for code legibility.

Reply:
remove this, clear from code change